custom/icds_reports/tests/agg_tests/outputs/agg_awc_2017-05-01_1_sorted.csv

```diff
@@ -1,4 +1,4 @@
+st1,All,All,All,All,2017-05-01,26,356,,,,,,,,,,,,,,,,,26,,317,475,,122,43,,,,,,,,70,87,618,9,3,46,22,0,1,3,0,5,12,1,2,3,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,17,8,12,,5,5,9,17,0,0,1,yes,,,,,,1,1,1,2,4,10,1322,421,431,,70,87,618,17,5,17,5,13,,,,17,192,775,1,1,0,0,1,0,0,0,0,0,0,2,3,185,94,51,4,157,37,,,13,12,14,0,0
+st2,All,All,All,All,2017-05-01,24,294,,,,,,,,,,,,,,,,,24,,378,513,,154,103,,,,,,,,85,79,668,6,20,65,14,0,0,4,1,4,4,1,1,16,2,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,12,7,9,,4,5,11,12,0,3,4,yes,,,,,,1,1,2,2,4,11,1476,525,531,,85,79,668,17,8,17,8,11,,,,13,154,832,0,0,0,1,0,0,0,0,0,0,1,6,0,193,108,82,0,164,78,,,12,10,11,0,0
+st7,All,All,All,All,2017-05-01,1,,,,,,,,,,,,,,,,,,,,0,1,,0,0,,,,,,,,0,1,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,yes,,,,,,1,1,1,1,1,1,1,0,4,,0,1,1,0,0,4,0,,,,,0,0,2,0,0,0,0,0,1,0,0,0,0,0,0,0,1,0,0,,1,0,,,,,0,3,2
```

Comment:
This commit is to test the change.
here. I have introduced 4 person cases in fixtures out of which 1 is migrated and 2 are out of school.